flybywiresim/aircraft
# fix(mcdu): maximum message length

## Summary of Changes

Messages are now writing directly to the scratchpad instead of passing the messages through the input validation.

This allows messages to be displayed again that are 24 chars long. (e.g. `WAIT FOR SYSTEM RESPONSE`)





## References

#4335 as well as [#private-references](https://discord.com/channels/738864299392630914/748096886531162123/896789211280211968)




## Additional context



Discord username (if different from GitHub): DerL30N#3751

## Testing instructions

- Check messages with up to 24 chars are being displayed e.g. `WAIT FOR SYSTEM RESPONSE` when selecting a system from the MCDUs MAIN MENU.
- Check user can only write up to 22 chars onto the scratchpad as per #4335


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
